### DISCUSSION TITLE
Help me with Pipewire

### DISCUSSION THREAD

**Discussion by sergeybutakov** [mikebrady/shairport-sync#1837] [General]
Hi 👋
Pipewire is selected by default, in raspi-config
What do I need to do to get sound? I'm using a docker image with a personal config
Please help me find how to run this 🥹

**Comment by sergeybutakov** [comment]
thnk

**Comment by sergeybutakov** [comment]
no, I didn't succeed😅
I don’t understand why there is sound in the system, but not in the container 🤯
Tell me how to run this, I've already tried everything.

**Comment by mikebrady** [comment]
Thanks for the post, but you should really [open an Issue](https://github.com/mikebrady/shairport-sync/issues/new/choose) about this.
